administration/src/bp-modules/stores/StoresCSVInput.tsx

```diff
@@ -40,6 +41,16 @@ const lineToStoreEntry = (line: string[], headers: string[], fields: StoreFieldC
   return new AcceptingStoreEntry(storeData, fields)
 }
 
+const hasStoreDuplicates = (stores: AcceptingStoreEntry[]) => {
+  return (
+    new Set(
+      stores.map(({ data }: { data: StoreData }) =>
+        JSON.stringify([data['name'], data['street'], data['houseNumber'], data['postalCode'], data['location']])
```

Comment:
another thought:
there is a clean() function in the backend that removes extra spaces from the data before saving it to the database.
so, we could load 2 stores with the names like "Store 1" and "Store   1" and they won't be considered duplicates here, but they will be considered in the backend.
maybe not critical, but a slight inconsistency.

Reply:
yes you are right. I will clean that in the frontend. Data should be imported as it is in the table view